src/studio/src/designer/frontend/ux-editor/containers/helpers/dnd-types.ts

```diff
@@ -0,0 +1,32 @@
+export interface EditorDndItem {
+  id: string;
+  containerId?: string;
+  onDrop?: (containerId: string, position: number) => void;
+  index?: number;
+  type: ItemType;
+}
+
+export enum ItemType {
+  TOOLBAR_ITEM = 'TOOLBAR_ITEM',
+  ITEM = 'ITEM',
+  CONTAINER = 'CONTAINER',
+}
+
+export enum ContainerPos {
+  TOP = 'TOP',
+  BOTTOM = 'BOTTOM',
+}
```

Comment:
Enum keys should be `PascalCase`, ref https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

```suggestion
export enum ItemType {
  ToolbarItem = 'TOOLBAR_ITEM',
  Item = 'ITEM',
  Container = 'CONTAINER',
}

export enum ContainerPos {
  Top = 'TOP',
  Bottom = 'BOTTOM',
}
```